handlers/bulk_refunds.go

```diff
@@ -15,13 +21,43 @@ func HandleGovPayBulkRefund(w http.ResponseWriter, req *http.Request) {
 
 	log.InfoR(req, "start POST request for bulk refunds")
 
-	_, _, err := req.FormFile("file")
+	file, _, err := req.FormFile("file")
 	if err != nil {
 		log.ErrorR(req, fmt.Errorf("error reading file from request: %w", err))
 		m := utils.NewMessageResponse("error reading file from request")
 		utils.WriteJSONWithStatus(w, req, m, http.StatusInternalServerError)
 		return
 	}
+	defer file.Close()
```

Comment:
Put `file.Close()` into a separate internal function to enable the returned error to be logged